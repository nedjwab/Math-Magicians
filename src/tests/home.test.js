import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/home';

it('renders correctly', () => {
  const home = renderer
    .create(<Home />)
    .toJSON();
  expect(home).toMatchSnapshot();
});
