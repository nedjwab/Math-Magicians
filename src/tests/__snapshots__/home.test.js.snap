// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="home-container"
>
  <img
    alt="gif"
    className="brain-img"
    src="https://upload.wikimedia.org/wikipedia/commons/e/e2/Math_-_Idil_Keysan_-_Wikimedia_Giphy_stickers_2019.gif"
  />
  <div
    className="body"
  >
    <h2>
      Welcome to our page!
    </h2>
    <div
      className="contenu"
    >
      While it may seem like math problems like the above have no real use in life, this couldnâ€™t be farther from the truth! Math is incredibly important in our lives and, without realizing it, we use mathematical concepts, as well as the skills we learn from doing math problems, every day. The laws of mathematics govern everything around us, and without a good understanding of them, one can encounter significant problems in life.
    </div>
  </div>
</div>
`;
